tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -29,6 +31,10 @@ spec:
       description: Path to the JSON file containing the mapped Snapshot spec in the data workspace
       type: string
       default: "snapshot_spec.json"
+    - name: dataPath
```

Comment:
Shouldn't you also change the rh pipeline to provide this param? Or will that be a second PR?

Reply:
Good call, I forgot

Reply:
fixed